infra/blueprint-test/pkg/gcloud/gcloud.go

```diff
@@ -79,6 +79,20 @@ func newCmdConfig(opts ...cmdOption) (*CmdCfg, error) {
 
 // RunCmd executes a gcloud command and fails test if there are any errors.
 func RunCmd(t testing.TB, cmd string, opts ...cmdOption) string {
+	op, err := RunCmdE(t, cmd, opts...)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return op
+}
+
+// check if the text comply to the output of a valid terraform vet execution
+func resourcesValidated(errText string) bool {
```

Comment:
Lets inline this as its only used within that func.

Reply:
fixed